cmd/torusd/main.go

```diff
@@ -110,10 +113,34 @@ func configureServer(cmd *cobra.Command, args []string) {
 		fmt.Fprintf(os.Stderr, "error parsing read-cache-size: %s\n", err)
 		os.Exit(1)
 	}
-	size, err = humanize.ParseBytes(sizeStr)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "error parsing size: %s\n", err)
-		os.Exit(1)
+	if strings.Contains(sizeStr, "%") {
+		var stat syscall.Statfs_t
```

Comment:
There's actually probably a better dependency here https://github.com/ricochet2200/go-disk-usage which does the same thing but brings less with it.